packages/core/src/components/popover/popover.stories.ts

```diff
@@ -0,0 +1,69 @@
+import { html, htmlMatrix, Meta, Story } from '../../../../../docs';
+import { Button } from '../button/button.story';
+import { Popover, PopoverProps } from './popover.story';
+
+const align = ['center', 'start', 'end'] as const;
+const position = ['top', 'left', 'right', 'bottom'] as const;
+
+export default {
+  title: 'Core/Popover',
+  component: Popover,
+  argTypes: {
+    align: {
+      control: { type: 'inline-radio', options: align },
+      defaultValue: 'center',
+      table: {
+        defaultValue: { summary: 'center' },
+        type: { summary: align.join('|') },
```

Comment:
Because https://github.com/onfido/castor/pull/882 was merged, please could you switch here (and in other places) to [use `optionsToSummary`](https://github.com/onfido/castor/pull/882/files#diff-28c6eb9281509786c4eec4fd1c77134e899e182aa1abfcd90e8b4168d3fea16dR10).

Reply:
Done.